mpyg321/mpyg321.py

```diff
@@ -188,10 +188,12 @@ def set_version_and_get_player(self, player):
             raise MPyg321NoPlayerFoundError("""No suitable player found""")
         return valid_player
 
-    def set_player(self, player):
+    def set_player(self, player, audiodevice):
         """Sets the player"""
         player = self.set_version_and_get_player(player)
         args = "--remote" if self.player_version == "mpg123" else "-R test"
+        if audiodevice:
+            args += " --audiodevice " + audiodevice
         self.player = pexpect.spawn(str(player) + " " + args)
```

Comment:
What happens when the audiodevice doesn't exist? Maybe we should pexpect for this kind of error and handle it? What do you think?

Reply:
OK, I feel like we can take care of this in another version, I'll merge this PR in the meantime (and we already have an open issue for #12